packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js

```diff
@@ -422,6 +423,200 @@ QUnit.module('General', () => {
 });
 
 QUnit.module('Tab select action', () => {
+    QUnit.module('onSelectionChanging', {
+        beforeEach() {
+            this.onSelectionChangingSpy = sinon.stub();
+            this.onSelectionChangedSpy = sinon.stub();
+            const initialOptions = {
+                items: [
+                    { text: '0' },
+                    { text: '1' },
+                    { text: '2' },
+                    { text: '3' }
+                ],
+                onSelectionChanging: this.onSelectionChangingSpy,
+                onSelectionChanged: this.onSelectionChangedSpy,
+                selectByClick: true,
+                selectedIndex: 0
+            };
+            this.init = (options) => {
+                this.$tabs = $('#tabs');
+                this.tabInstance = this.$tabs
```

Comment:
Let's rename to make the tests more compact? Some rows are really long now
```suggestion
                this.tabs = this.$tabs
```